src/wrapper/jnienv.rs

```diff
@@ -986,6 +949,8 @@ impl<'a> JNIEnv<'a> {
     }
 
     /// Unpin the array returned by `get_string_utf_chars`.
+    #[allow(clippy::not_unsafe_ptr_arg_deref)]
+    /// It is safe to dereference a pointer that comes from `get_string_utf_chars`.
```

Comment:
I believe this comment should not be visible in docs. And I'd place it above lint, not below.

Reply:
Yep, that was definitely the intention of an implementation comment.

Reply:
Guys, I feel confused 🤔. The first version I made looked like this:
```
    /// Unpin the array returned by `get_string_utf_chars`.
    ///
    /// It is safe to dereference a pointer that comes from `get_string_utf_chars`.
    #[allow(clippy::not_unsafe_ptr_arg_deref)]
```
And then I changed it twice AFAIR. And now it looks to me that @vitvakatu suggests the initial version.
Could you please give me a sample of comments that we all happy with? Maybe here in comments as code suggestion.

Reply:
I suppose it should look the following way:
```rust
/// Unpin the array returned by `get_string_utf_chars`.
// It is safe to dereference a pointer that comes from `get_string_utf_chars`.
#[allow(clippy::not_unsafe_ptr_arg_deref)]
pub fn release_string_utf_chars(&self, obj: JString, arr: *const c_char) -> Result<()> {}
```
Note: the second comment is regular (`//`), not the documentation one (`///`).

Reply:
Code suggestions sucks with multiple lines, but it should be like this:
```
/// Unpin the array returned by `get_string_utf_chars`.
///
// It is safe to dereference a pointer that comes from `get_string_utf_chars`.
#[allow(clippy::not_unsafe_ptr_arg_deref)]
```

It was:
```
/// It is safe to dereference a pointer that comes from `get_string_utf_chars`.
#[allow(clippy::not_unsafe_ptr_arg_deref)]
/// Unpin the array returned by `get_string_utf_chars`.
```

Reply:
Great, thanks for clarification! 👍